src/main.ts

```diff
@@ -0,0 +1,118 @@
+import * as core from '@actions/core'
+import * as util from 'util'
+import { ExecOptions } from '@actions/exec/lib/interfaces';
+import { ToolRunner } from '@actions/exec/lib/toolrunner';
+import { getTrivy } from './trivyHelper'
+import { getDockle } from './dockleHelper'
+const download = require('download');
+
+let trivyEnv: { [key: string]: string } = {};
+let dockleEnv: { [key: string]: string } = {};
+
+async function getWhitelistFileLoc(whitelistFilePath: string, whitelistFileBranch: string): Promise<string> {
+    const whitelistFileUrl = `https://raw.githubusercontent.com/${process.env.GITHUB_REPOSITORY}/${whitelistFileBranch}/${whitelistFilePath}`;
+    const whitelistFilePathParts = whitelistFilePath.split('/');
+    const whitelistFileName = whitelistFilePathParts[whitelistFilePathParts.length - 1];
+    const whitelistFileDownloadDir = `${process.env['GITHUB_WORKSPACE']}/_temp/containerScanWhitelist`;
```

Comment:
Add something unique to handle multiple usage of this action on the runner.
`${process.env['GITHUB_WORKSPACE']}/_temp/containerScan_${date}/whitelist`

[Refer](https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/Common/docker-common-v2/containerconnection.ts#L333) this.

Reply:
Added